README.md

```diff
@@ -88,57 +88,24 @@ function resultHandler(results) {
 
 ## Usage as a Node Worker
 ```typescript
-import { Worker, MessageChannel, MessagePort } from 'worker_threads';
-import { deserializeSuggestedItems } from './utils';
+const { SpellcheckerWasm } = require('../lib/nodejs/SpellcheckerWasm.js');
 
-const wasmPath = require.resolve('spellchecker-wasm/lib/spellchecker-wasm.wasm');
-const dictionaryLocation = require.resolve('spellchecker-wasm/lib/frequency_dictionary_en_82_765.txt');
+const wasmPath = resolve(__dirname, '../lib/spellchecker-wasm.wasm');
+const dictionaryPath = resolve(__dirname, '../lib/frequency_dictionary_en_82_765.txt');
```

Comment:
These examples are setup assuming an installed library in `node_modules`. Using `__dirname` here assumes a cloned repo. Reverting this back to using `require.resolve()` should keep it consistent.

Reply:
@Devmacs - ^^^ Let's get this merged in! Are you able to make the updates I suggested?

Reply:
Sorry for the delay, it's been quite busy over here. I'll take care of this later today; thanks for the reminder.

Reply:
Done. Please let me know if there's anything else you'd like me to change. Thanks!